<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>windowicon.png</normaloff>windowicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>51</width>
      <height>93</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_velocity_more">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_velocity_less">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_perspective_left">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>49</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_perspective_right">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>49</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_perspective_down">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>49</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_perspective_up">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>49</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>781</width>
      <height>381</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>51</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButton_control_play">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>▶</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_control_pause">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>||</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_control_stop">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>▉</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_home">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ξ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_next_motion">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>⎇</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_lock">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>⇄</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>0</y>
       <width>41</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QPushButton" name="pushButton_axis_run_a1">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>A1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_axis_run_a2">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>A2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_axis_run_a3">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>A3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_axis_run_a4">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>A4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_axis_run_a5">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>A5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_axis_run_a6">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>A6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_axis_run_a7">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>A7</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_axis_step_motion">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>⥁</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_jog">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>40</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Prog:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>JOG:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_information">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_prog">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>79</x>
       <y>69</y>
       <width>631</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTextEdit" name="textEdit_program"/>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_log"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>151</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>51</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_perspective_up_2">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>↑</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_perspective_down_2">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>↓</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>40</y>
      <width>151</width>
      <height>361</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_axis_6_less">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>260</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_axis_1_less">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_axis_5_more">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_axis_2_less">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_axis_1_more">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_axis_3_less">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_axis_3_more">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_axis_5_less">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>210</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_axis_4_less">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_axis_4_more">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_axis_6_more">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_axis_2_more">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_axis_7_more">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_axis_7_less">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>310</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>781</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_move_1">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>M1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_move_2">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>M2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_move_3">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>M3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_move_4">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>M4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_move_5">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>M5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_move_6">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>M6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_move_7">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>M7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_move_8">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>M8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_move_9">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>M9</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_move_10">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>M10</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>frame_3</zorder>
   <zorder>frame_2</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>pushButton_perspective_left</zorder>
   <zorder>pushButton_perspective_right</zorder>
   <zorder>pushButton_perspective_down</zorder>
   <zorder>pushButton_perspective_up</zorder>
   <zorder>frame</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1129</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
